database/redis/database.go

```diff
@@ -92,15 +128,23 @@ func newPoolDialer(logger moira.Logger, config Config) PoolDialer {
 				DialTimeout:       dialTimeout,
 			},
 		)
+		if !config.AllowSlaveReads {
+			return sentinelDialer, nil
+		}
+		logger.Info("Redis: Sentinel slaves pooling enabled")
+		slaveDialer := NewSentinelSlavePoolDialer(sentinelDialer)
+
+		return sentinelDialer, slaveDialer
 	}
 
 	serverAddr := net.JoinHostPort(config.Host, config.Port)
 	logger.Infof("Redis: %v, DB: %v", serverAddr, config.DB)
-	return &DirectPoolDialer{
+	mainDialer = &DirectPoolDialer{
 		serverAddress: serverAddr,
 		db:            config.DB,
 		dialTimeout:   dialTimeout,
 	}
+	return
```

Comment:
I feel that named returns make code less clear in this case. You return explicit values 2 times out of 3 (lines 132 and 137), and named return values just once in the end. It takes extra time to follow the code and understand where did these values come from. I suggest returning explicit values everywhere.

Reply:
Done